profile/README.md

```diff
@@ -0,0 +1,23 @@
+<h2 align="center">JSON Schema is a declarative language that allows you to annotate and validate JSON documents</h2>
+
+[![Slack](https://img.shields.io/static/v1?label=Slack&message=@json-schema&color=yellow)](https://asyncapi.slack.com)
+[![Twitter](https://img.shields.io/static/v1?label=Twitter&message=@jsonschema&color=9cf)](https://twitter.com/jsonschema)
+[![LinkedIn](https://img.shields.io/static/v1?label=LinkedIn&message=@jsonschema&color=lightgray)](https://www.linkedin.com/company/jsonschema)
+[![YouTube](https://img.shields.io/static/v1?label=Youtube&message=@JSONSchemaOrgOfficial&color=red)](https://www.youtube.com/@JSONSchemaOrgOfficial)
+[![Google Groups](https://img.shields.io/static/v1?label=Google&nbsp;Groups&message=@json-schema&color==blueviolet)](https://groups.google.com/g/json-schema)
+
+### 👋 Welcome
+Welcome to JSON Schema, a declarative language that allows you to annotate and validate JSON documents.
+
+JSON Schema is part of [the OpenJS Foundation](https://openjsf.org/blog/2022/01/31/json-schema-joins-openjs-foundation/).
+
+### 🌎 Join the JSON Schema Community
+To get involved with our Community, please make sure you are familiar with the project's [Code of Conduct](https://github.com/json-schema-org/.github/blob/main/CODE_OF_CONDUCT.md).
+
+- **Join our [Slack workspace](https://json-schema.org/slack)**: The best place to ask questions, learn, get help, or discuss all things JSON Schema.
+- **Join our public meetings**: We hold monthly [Office Hours](https://github.com/orgs/json-schema-org/discussions/34) and weekly [Open Community Working Meetings](https://github.com/orgs/json-schema-org/discussions/35). Please join!
+- **Follow our [YouTube Channel](https://www.youtube.com/watch?v=48S8-GwRh-g&list=PLHVhS4Tj1YZPYt6sMkvf4nW8zKvZExVA4)**: To find the recordings of our public community meetings and JSON Schema learning resources.
+
+### 👩🏽‍💻 Contributing to JSON Schema
+
+JSON Schema is always looking for new contributions. From documentation, tooling, website, or specification development; Any contribution is valued and welcome. Are you interested in contributing? Please read our [guidelines for contributing](CONTRIBUTING.md).
```

Comment:
There's currently no contributing.md file. Was this intended to link to one in another repo?

The contributing.md file in the spec repo is widly outdated. We could probably do with adding a notice to the top of it noting that it is outdated.

Reply:
My suggestion is start pointing to contributing.md file in the spec repo, and replace it in the next days with a new contributing at org level. Refining the contributor journey is something we agreed in the Team Summit.

Reply:
I suggest creating the contribution file first, then creating the links to it.

Reply:
My preferance would be to invite people to use the Discussions and Slack server, until we have an up to date contributing guide document.
However, I won't block on this. We can itterate.

Reply:
@gregsdennis do you have a strong preferenace to make sure the contributing file is up to date first? Or shall we merge this and open new Issues?

Reply:
I just thought we should have _a_ contribution file before we linked to it.  If we have one, that's fine; it can be updated later.